openlcs/openlcs/settings.py

```diff
@@ -311,12 +321,3 @@
         },
     },
 }
-
-try:
-    # pylint:disable=wildcard-import,unused-wildcard-import
-    parent_dir = os.path.abspath(os.path.dirname(__file__))
-    if parent_dir not in sys.path:
-        sys.path.append(parent_dir)
-    from settings_local import *  # noqa
-except ImportError:
-    pass
```

Comment:
Is this a revert of https://github.com/RedHatProductSecurity/openlcs/pull/131/?

Reply:
YES

Reply:
I was asking the change and I saw there have been several MRs related to the change, now a new revert, make sure the update aligns to your discussions.